### ISSUE TITLE
Hunter - Pet - "Wind Serpent" class Lightning Breath Rank 2 does the same damage as Rank 1 

### ISSUE THREAD

**Issue by scormonel69** [chromiecraft/chromiecraft#542]
#### WHAT CLIENT DO YOU PLAY ON?

- [ ] enGB
- [x] enUS
- [ ] other (specify)

##### FACTION

- [ ] Alliance
- [x] Horde

##### CONTENT PHASE:

- [x] Generic
- [ ] 1-19
- [ ] 20-29
- [ ] 30-39
- [ ] 40-49
- [ ] 50-59

##### CURRENT BEHAVIOUR:

SpellID: 25008 from pet creature type - Wind Serpent does the same damage as the spell from rank 1.

##### EXPECTED BLIZZLIKE BEHAVIOUR:

Lightning Breath LVL 1 - 12 - 14 dmg
Lightning Breath LVL 2 - 21 - 23 dmg
Lightning Breath LVL 3 - 36 - 40 dmg
Lightning Breath LVL 4 - 51 - 69 dmg
Lightning Breath LVL 5 - 78 - 90 dmg
Lightning Breath LVL 6 - 99 - 113 dmg
 

##### STEPS TO REPRODUCE THE PROBLEM:

1.  Have a Hunter char at least lvl 20
2.  Go to Barrens - south of camp taurajo
3.  Tame a Thunderhawk Cloudscraper
4. Observe the Lightning Breath skill, rank and damage

##### EXTRA NOTES:

More details can be found in comments at : https://classic.wowhead.com/npc=3472/washte-pawne

![WoWScrnShot_050521_191108](https://user-images.githubusercontent.com/82616893/117173713-b7e7e780-add5-11eb-83c7-e5b88808559e.jpg)

##### AC HASH/COMMIT:

https://github.com/chromiecraft/azerothcore-wotlk/commit/91081f4ad8fed98d7bdc9c0ae13d05c86032df9f

##### OPERATING SYSTEM:

Ubuntu 20.04

##### MODULES:

- [mod-cfbg](https://github.com/azerothcore/mod-cfbg)
- [mod-duel-reset](https://github.com/azerothcore/mod-duel-reset)
- [mod-desertion-warnings](https://github.com/azerothcore/mod-desertion-warnings)
- [mod-ah-bot](https://github.com/azerothcore/mod-ah-bot)
- [mod-eluna-lua-engine](https://github.com/azerothcore/mod-eluna-lua-engine)
- [mod-server-auto-shutdown](https://github.com/azerothcore/mod-server-auto-shutdown)
- [lua-LevelUpReward](https://github.com/55Honey/Acore_LevelUpReward)
- [lua-CarbonCopy](https://github.com/55Honey/Acore_CarbonCopy)
- [mod-pvp-titles](https://github.com/azerothcore/mod-pvp-titles)

##### OTHER CUSTOMIZATIONS:

None.

##### SERVER:

ChromieCraft

**Comment by Azcobu** [comment]
Thanks for the ticket. However, I think this is likely a client display bug. I levelled a hunter to 20 and tamed a Thunderhawk Cloudscraper, which gave lvl 1 Lightning Breath, and tested against a target dummy. I then used GM commands to give the pet lvl 2 Breath, and fired it at the dummy again. This kept the hunter level the same both times, meaning the only thing that changed was the Lighting Breath rank.

The result was rank 1 averaged 9.5 dmg (excluding the crit) and rank 2 averaged 16 dmg. (Bear in mind the pet was unhappy, so was only doing 75% of normal damage.) So this seems in line with your table above.

![WoWScrnShot_050621_232142](https://user-images.githubusercontent.com/81782124/117310665-a3354d80-aec2-11eb-84e1-136c7f55a7ec.jpg)

So this suggests to me that your pet's spell is scaling correctly, and the spellbook just isn't displaying the spell rank damage correctly.
